test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/StoreTransactionTests.cs

```diff
@@ -26,38 +31,31 @@ namespace Microsoft.Health.Dicom.Web.Tests.E2E.Rest;
 
 public abstract class StoreTransactionTests : IClassFixture<HttpIntegrationTestFixture<Startup>>, IAsyncLifetime
```

Comment:
In this class, I am trying to remove instances of buffering the file to a `MemoryStream` and instead serialize the content directly to the network stream via `HttpContent` like `DicomContent`.